packages/openid4vc/src/openid4vc-issuer/router/verifyResourceRequest.ts

```diff
@@ -27,6 +36,15 @@ export async function verifyAccessToken(
     },
   })
 
+  const fullUrl = request.protocol + '://' + request.get('host') + request.originalUrl
+  await verifyResourceDPoP(
```

Comment:
I know it looks a bit weird, but consistency is king imo

```suggestion
  await verifyResourceDpop(
```